index.html

```diff
@@ -23,8 +23,55 @@ <h3>Lorem Ipsum</h3>
       <h6>Lorem Ipsum</h6>
     </div>
   </div>
+  <div>
+    <h4>Subscribe to our newsletter</h4>
+    <input class="input-newsletter" type="text" placeholder="Enter your e-mail">
+    <button type="submit" name="action">Submit
+    </button>
+</div>
+<h2>NEWS</h2>
+<div class ="flexbox">
+  <div class ="box1">
+    <div class="news1">
+    <img  class = "img1" src = https://www.uni-hohenheim.de/fileadmin/uni_hohenheim/Studiengaenge/WiSo/Master/finance-msc/msc-finance-start.jpg>
+    </div>
+    <div class= "content"></div>
+    <h5><a href="https://finance.yahoo.com/">Avoid 'knee-jerk' investing decisions amid election uncertainty: financial adviser</a> </h5>
+  </div>
+ </div>
+</div>
+<div class ="flexbox">
+  <div class ="box1">
+    <div class="news1">
+    <img  class = "img1" src = https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcSdT1WHrAln78VgfXjY2IH0vy7YjlAUNlD1aw&usqp=CAU>
```

Comment:
classes should be named sth you can easily recognize later. You don't have to change this now but consider using a more meaningful name. In the future you want any developer reading your CSS to be able to immediatly understand what each class may be referring to ;)